config/jobs/kubernetes-sigs/cluster-api-provider-digitalocean/cluster-api-provider-digitalocean-presubmits.yaml

```diff
@@ -37,13 +37,28 @@ presubmits:
     always_run: true
     decorate: true
     path_alias: sigs.k8s.io/cluster-api-provider-digitalocean
+    labels:
+      preset-service-account: "true"
+      preset-bazel-scratch-dir: "true"
+      preset-bazel-remote-cache-enabled: "true"
     spec:
       containers:
-      - image: gcr.io/k8s-testimages/kubekins-e2e:v20181024-be2f242dd-experimental
-        command:
-        - runner.sh
-        - kubernetes_bazel.py
```

Comment:
@krzyzacy would you know what needs to be changed to get this working with pod utilities?

Reply:
ooops missed that in review

ref https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-testing/kubetest-canaries.yaml#L96-L97

kubernetes_bazel.py is not in path, so, `/workspace/scenarios/kubernetes_bazel.py` it is.

and you do need the bazel presets

Reply:
Updated. Ptal, thanks!